apps/reflect/src/forms/TwitchBotForm.tsx

```diff
@@ -117,6 +126,43 @@ const TwitchBotForm = () => {
               </FormControl>
             )}
           />
+        </Box>
+        <Box
+          sx={{
+            "& > :not(style)": {
+              m: 1,
+            },
+          }}>
```

Comment:
let's rethink about this, i think there are some confusions here that we can solve with less start ways :D